.github/workflows/qa.yml

```diff
@@ -12,6 +12,7 @@ jobs:
       - name: PHP-CS-Fixer
         uses: docker://jakzal/phpqa:1.80.0-php7.4-alpine
         with:
+          entrypoint: ./.phpqa_entrypoint.sh
```

Comment:
```suggestion
```

We can remove this entrypoint and add a step between checkout and PHP-CS-Fixer:

```
            -   name: Delete composer files to avoid installing dependencies
                run: rm -v composer.json composer.lock
```

(PHP-CS-Fixer doesn't need the dependencies to be installed)